docs/src/manifolds/rotations.md

```diff
@@ -0,0 +1,7 @@
+# Rotations
+
```

Comment:
In the long run I would prefer to have a general explanation and maybe an illustration and applications in the doc (like explaining signs do...in a museum)

Reply:
I totally agree, we need good docs here.